app/helpers/hold_helper.rb

```diff
@@ -28,4 +30,21 @@ def filter_holds_by_source(holds)
 
     holds
   end
+
+  # This patches a PaperTrail bug that saves enums to the version table as integers instead of their mapped values.
+  # We never noticed this bug because it occurs only with YAML data, but PT is able to correctly parse enums when it
+  # unserializes YAML data. This is not the case with JSON data, so historical statuses are stored as integers and
+  # render as integers.
```

Comment:
`render as integers unless we handle them with this helper` (or something like that clarifying that this resolves the problem).

Reply:
Let me know if the commit I just pushed addresses this.

I'd also like to document this somewhere in the readme, likely within an overview of our PT implementation. I think that can wait until we decide how permanent this is.